sceptre/cli/list.py

```diff
@@ -89,6 +93,39 @@ def list_outputs(ctx, path, export):
                         ),
                         "text",
                     )
+
+    # Format outputs as !stack_output references.
+    elif export == "stackoutput":
```

Comment:
what are you thoughts about changing this to `stack_output` to match `!stack_output`?